code/modules/research/xenoarchaeology/finds/finds.dm

```diff
@@ -525,7 +525,6 @@
 			possible_spawns += /obj/item/clothing/mask/morphing
 			possible_spawns += /obj/item/clothing/mask/morphing/amorphous
 			//possible_spawns += /obj/item/clothing/mask/happy PENDING REWORK
-			//possible_spawns += /obj/item/clothing/mask/stone WHEN I CODE IT
```

Comment:
I ended commiting a non-updated version of the file then fixed it :man_shrugging: